### PR TITLE
#355 Added test for Row component

### PR DESCRIPTION
**Pull request by tannakartikey**
**Description of PR**
Added the spec for `src/components/row.js`

**Type of PR**

- [ ] Bugfix
- [x] New feature

**Relevant Issues**  
#355 

**Checklist**

- [ ] Compiles and passes lint tests
- [ ] Properly formatted
- [ ] Tested on desktop
- [ ] Tested on phone

**Screenshots**

### CODE REVIEW

File: src/__tests__/components/Row.test.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    expect(confirmed).toEqual('11');
+    expect(active).toEqual('1');
+    expect(recovered).toEqual('10');
+    expect(deaths).toEqual('-');
+  });
+
+  test('State last update data', () => {
+    const stateLastUpdate = wrapper.find('tr.state-last-update');
+
+    expect(stateLastUpdate.text()).toMatch(/14 days ago/i);
```
Comment (shuklaayush) [inline src/__tests__/components/Row.test.js:72]: Won't this fail from tomorrow? The corresponding code in row.js calculates the days passed based on the current date.

Comment (tannakartikey) [reply src/__tests__/components/Row.test.js:72]: @shuklaayush the data is hardcoded in the spec:
https://github.com/covid19india/covid19india-react/blob/cd9da8927100ed44b6d6fe6f58e1ff6f2e391501/src/__tests__/components/Row.test.js#L12

Comment (shuklaayush) [reply src/__tests__/components/Row.test.js:72]: No, I mean when the difference in dates is calculated inside row.js:
https://github.com/covid19india/covid19india-react/blob/584ec6fecb9cf27d6e79c37dbc0f9967963fbf36/src/components/row.js#L183-L186

Comment (tannakartikey) [reply src/__tests__/components/Row.test.js:72]: Oh, I see. Good catch, @shuklaayush 
I may have to stub the date.

Comment (tannakartikey) [reply src/__tests__/components/Row.test.js:72]: @shuklaayush done! Mocked the date: https://github.com/covid19india/covid19india-react/pull/1042/commits/f692cda4e2a4c3faf2b493fa6d5a275a1f70330c

File: src/components/row.js [hunk 2]
```diff
@@ -8,6 +8,19 @@ import {
 import {formatDistance} from 'date-fns';
 import {Link} from 'react-router-dom';
 
+function lastUpdate(props) {
+  const {lastupdatedtime: lastUpdatedTime} = props.state;
```
Comment (shuklaayush) [inline src/components/row.js]: I think it makes sense to directly input props.lastupdatedtime. No point in passing the whole thing

Comment (tannakartikey) [reply src/components/row.js]: @shuklaayush right. Done.

**Review by shuklaayush — APPROVED** [review]

**Pull request merged**